batch-hashtag-count/README.md

```diff
@@ -73,15 +67,14 @@ xd:> stream create --name tweets --definition "twitterstream \
 --accessTokenSecret='your_credentials' | hdfs --rollover=2M" --deploy
 ```
 
-or alternatively you can provide the credentials via:
-
-`config/modules/source/twitterstream/twitterstream.properties`
+or alternatively you can provide the credentials in `config/modules/modules.yml`
 
 ```
-consumerKey=<your consumer key>
-consumerSecret=<your consumer secret>
-accessToken=<your access token>
-accessTokenSecret=<your token secret>
+twitter:
+   consumerKey=<your consumer key>
```

Comment:
Should be formatted as consumerKey: <your consumer key> instead of consumerKey=<your consumer key>